src/models/subscription.test.ts

```diff
@@ -204,6 +237,22 @@ describe("Subscription", () => {
 				})]);
 			});
 		});
+		describe("findForRepoNameAndOwner", () => {
+			it("should fetch the subscription for the given repo name and owner", async () => {
+				const repo0Sub = await Subscription.findForRepoNameAndOwner("repo-0", "atlassian", jiraHost);
+				const repo00Sub = await Subscription.findForRepoNameAndOwner("repo-0-0", "atlassian", jiraHost);
+				const repo1Sub = await Subscription.findForRepoNameAndOwner("repo-1", "atlassian", jiraHost);
+				const repo2Sub = await Subscription.findForRepoNameAndOwner("repo-2", "atlassian", jiraHost);
+				const repoRandomSub = await Subscription.findForRepoNameAndOwner("repo-unknown", "atlassian", jiraHost);
```

Comment:
I think this needs to be a separate test, e.g.

`returns undefined when repo cannot be find` or something like that

Reply:
Separated in a new test case.